lessons/8-intermediate-r-2/lesson_script.R

```diff
@@ -0,0 +1,151 @@
+library(dplyr)
+
+### Copy this from Tuesday's script
+
+my_function <-
+  function(data_url = "https://github.com/cct-datascience/repro-data-sci/raw/r-lessons/lessons/7-intermediate-r-1/lesson-data/Biscoe.csv") {
+    island_dat <- read.csv(data_url)
+    
+    
+    island_dat <- island_dat |>
+      na.omit()
+    
+    
+    island_dat_means <- island_dat |>
+      group_by(species, sex) |>
+      summarize(across(ends_with("mm") | ends_with("g"), mean))
+    
+    island_dat_means_imperial <- island_dat_means |>
+      mutate(
+        across(ends_with("mm"), ~ . * 0.03937008, .names = "{.col}_in"),
+        across(ends_with("g"), ~ . * 0.002204623, .names = "{.col}_lb")
+      )
+    
+    island_dat_means_imperial <- island_dat_means_imperial |>
+      rename_with( ~ stringr::str_replace(., "mm_in", "in"), .cols = ends_with("mm_in")) |>
```

Comment:
Does this work?
```suggestion
      rename_with(\(x) stringr::str_replace(x, "mm_in", "in"), .cols = ends_with("mm_in")) |>
```
I think it does, and if it does it might be better to stay consistent with how we show anonymous functions since this is the way you use them in `map()` later on.

Reply:
This works!